api/src/main/java/com/velocitypowered/api/event/command/CommandExecutedEvent.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020-2023 Velocity Contributors
+ *
+ * The Velocity API is licensed under the terms of the MIT License. For more details,
+ * reference the LICENSE file in the api top-level directory.
+ */
+
+package com.velocitypowered.api.event.command;
+
+import com.google.common.base.Preconditions;
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.event.annotation.AwaitingEvent;
+
+/**
+ * This event is fired when velocity executed a command. This event is called after the event
+ * is handled. It's not fired for forwarded commands or commands with syntax errors.
+ */
+@AwaitingEvent
+public final class CommandExecutedEvent {
```

Comment:
I think the event needs a better name, anyone can get confused between `CommandExecuteEvent` and `CommandExecutedEvent`, especially if you autocomplete the event with an IDE

Reply:
Maybe `ServerCommandExecutedEvent`?

Reply:
> Maybe `ServerCommandExecutedEvent` or `PostCommandExecuteEvent`?

I like the `PostCommandExecuteEvent` more, but I am not totally convinced because the event is not executed after each command, but, after each command successfully executed from Velocity

Reply:
We can change the event to cover all result types (Forward, Syntax error and execution), then PostCommandExecuteEvent would be the correct name

Reply:
I renamed it to `PostCommandExecuteEvent` and refactored the code a little bit.

This way the result can catch an exception and syntax errors too.